frontend/components/forms/fields/Dropdown/DisabledOptionTooltipWrapper/DisabledOptionTooltipWrapper.tsx

```diff
@@ -0,0 +1,62 @@
+import classnames from "classnames";
+import React from "react";
+import { Tooltip as ReactTooltip5 } from "react-tooltip-5";
+
+import { uniqueId } from "lodash";
+
+interface IDisabledOptionTooltipWrapper {
+  children: React.ReactNode;
+  isDelayed?: boolean;
+  className?: string;
+  tooltipClass?: string;
+  clickable?: boolean;
+  tipContent: React.ReactNode;
+}
+
+const baseClass = "disabled-option-tooltip-wrapper";
+
+const DisabledOptionTooltipWrapper = ({
+  children,
+  tipContent,
+  isDelayed,
+  className,
+  tooltipClass,
+  clickable = true,
+}: IDisabledOptionTooltipWrapper) => {
+  const wrapperClassNames = classnames(baseClass, className);
+
+  const elementClassNames = classnames(`${baseClass}__element`);
+
+  const tipClassNames = classnames(
+    `${baseClass}__tip-text`,
+    `${baseClass}__dropdown-tooltip-arrow`,
+    tooltipClass
+  );
+
+  const tipId = uniqueId();
+
+  return (
+    <span className={wrapperClassNames}>
+      <div className={elementClassNames} data-tooltip-id={tipId}>
+        {children}
+      </div>
+      <ReactTooltip5
+        className={tipClassNames}
+        id={tipId}
+        delayShow={isDelayed ? 500 : undefined}
+        delayHide={isDelayed ? 500 : undefined}
+        place="left"
```

Comment:
While we're working on this I think it'd be a good idea to make this dynamically any of the 4 possible places with `"left"` default. Can also be for a future iteration so won't hold up the PR for it. We'd also have to include the core react tooltipo css styles (see below) for each direction and remove the `classNameArrow` prop.

```
[class*='react-tooltip__place-top'] > .styles-module_arrow__K0L3T {
  transform: rotate(45deg);
}
[class*='react-tooltip__place-right'] > .styles-module_arrow__K0L3T {
  transform: rotate(135deg);
}
[class*='react-tooltip__place-bottom'] > .styles-module_arrow__K0L3T {
  transform: rotate(225deg);
}
[class*='react-tooltip__place-left'] > .styles-module_arrow__K0L3T {
  transform: rotate(315deg);
}
```